lib/workos/user_management.rb

```diff
@@ -678,6 +651,32 @@ def send_password_reset_email(email:, password_reset_url:)
         WorkOS::UserAndToken.new(response.body)
       end
 
+      # Resets user password using token that was sent to the user.
+      # @param [String] token The token that was sent to the user.
+      # @param [String] new_password The new password to set for the user.
+      #
+      # @return WorkOS::User
+      sig do
+        params(
+          token: String,
+          new_password: String,
+        ).returns(WorkOS::User)
+      end
+      def reset_password(token:, new_password:)
```

Comment:
The `reset_password` method should be added to the [API reference](https://workos.com/docs/reference/user-management/password-reset/complete)